src/pwa-buildpack/project-setup/install-dependencies/index.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Install project dependencies
+---
+
+In the [previous topic], you created the basic directory structure and files for a Magento theme.
+In this topic you will initialize the `package.json` file and install project dependencies for your Magento PWA theme.
+
+## Initialize and define the package configuration file
+
+1. The following command uses `npm` to create your project's `package.json` file using default project values:
+   
+   ``` bash
+   npm init -y
+   ```
+   
+1. Run the following command to install [peregrine] and [React] as a production dependency:
+
+   ``` bash
+   npm install --save @magento/peregrine \
+   react react-dom react-redux react-router-dom redux
+   ```
+
+   These dependencies allow you to use the UI components and services provided by the peregrine project to develop your theme.
+
+1. Run the following command to install [pwa-buildpack] and its standard peer dependincies:
+
+   ``` bash
+   npm install --save-dev @magento/pwa-buildpack \
+   babel-core babel-loader babel-helper-module-imports \
+   babel-plugin-syntax-jsx babel-plugin-transform-class-properties \
+   babel-plugin-transform-object-rest-spread babel-plugin-transform-react-jsx \
+   webpack webpack-cli webpack-dev-server
```

Comment:
This needs to change to `webpack@3 webpack-dev-server@2` (no `webpack-cli` for webpack 3!